src/PresentationalComponents/Header/Header.test.js

```diff
@@ -100,20 +101,20 @@ describe('Header component', () => {
                 }
             ]
         };
-        const wrapper = withLink(<Header {...header} />);
-        expect(toJson(wrapper)).toMatchSnapshot();
+        const { asFragment } = withLink(<Header {...header} />);
+        expect(asFragment()).toMatchSnapshot();
     });
-
-    it('should render with tabs only, click a tab', () => {
+    //NOTE: Should be rewritten because of wrapper changes
```

Comment:
Because of what changes?

Reply:
I formulated that incorrectly, due to how RTL works we shouldn't test the history changes from the inside the component. If we want to test it we should do it separately https://stackoverflow.com/questions/70313688/how-i-could-test-location-with-memoryrouter-on-react-router-dom-v6